Module4_Labs/Lab5_nColorable/Check Point/testcases_checkpoint.py

```diff
@@ -0,0 +1,161 @@
+
```

Comment:
Adjust the checkpoints slightly to reflect the code in ncolorablefinal.py. Otherwise, they look great!!